### ISSUE TITLE
After login, the user is redirected to https://domainsample.com/menu_links

### ISSUE THREAD

**Issue by exetico** [vinkas0/discourse-navigation#13]
After login, the user is redirected to https://domainsample.com/menu_links. (aka forum.com/menu_links aka {domain}/menu_links)

Could it be anyting going on with ths plugin? We're running a private forum, but i can't find any details of why this i happening.

**Comment by KazWolfe** [comment]
Can confirm this. It doesn't seem to be reproducible 100% of the time, but it is definitely annoying.

**Comment by arnelbh** [comment]
It is happening only when "Login Requiered" in admin section is turned on. If user wants to use a forum, must log in, than after successfull log in, the discourse redirects to the forumdomainexample.org/menu_links like page...

**Comment by exetico** [comment]
You're right @arnelbh  - The question is, if it's a issue with discourse-navigation, or if the issue actually are within Discourse? I'm not quiet sure - and would like to know if the dev of discourse-navigation have any informations about this - or are able to debug :-)

Update:
I can see that [Vinoth told that it's not in active development anymore](https://meta.discourse.org/t/login-required-activated-after-successfull-login-it-transfers-you-to-nonexisting-page/86825)...

Guess i will go with a [simple hack](https://meta.discourse.org/t/adding-a-guidelines-link-to-the-hamburger-menu/56857/3?u=exetico) for now.
```
<script type="text/discourse-plugin" version="0.4">
  api.decorateWidget('hamburger-menu:footerLinks', () => {
    return [
        { href: 'https://domain.com/url1', rawLabel: 'Title1' }, 
        { href: 'https://domain.com/url2', rawLabel: 'Title2' }
    ];
  })
</script>
```
